deploy/helm/sumologic/values.yaml

```diff
@@ -2130,6 +2132,151 @@ otellogs:
   ##             - linux
   additionalDaemonSets: {}
 
+## Experimental
+## Configure OpenTelemetry Logs Collector for Windows Nodes
+otelwindows:
```

Comment:
Maybe `otellogswindows`? That'd be more consistent with our current naming for these sections.